include/ultra64/rcp.h

```diff
@@ -39,6 +39,24 @@
 #define VI_X_SCALE_REG          0x04400030  //VI x-scale
 #define VI_Y_SCALE_REG          0x04400034  //VI y-scale
 
+// VI_CONTROL_REG (read) bits
+#define VI_CTRL_TYPE_16             0x00002 // [1:0] pixel size: 16 bit
+#define VI_CTRL_TYPE_32             0x00003 // [1:0] pixel size: 32 bit
+#define VI_CTRL_GAMMA_DITHER_ON     0x00004 // 2: default = on
+#define VI_CTRL_GAMMA_ON            0x00008 // 3: default = on
+#define VI_CTRL_DIVOT_ON            0x00010 // 4: default = on
+#define VI_CTRL_SERRATE_ON          0x00040 // 6: on if interlaced
+#define VI_CTRL_ANTIALIAS_MASK      0x00300 // [9:8] anti-alias mode
+#define VI_CTRL_ANTIALIAS_MODE_0    0x00000 // Bit [9:8] anti-alias mode: AA enabled, resampling enabled, always fetch extra lines
+#define VI_CTRL_ANTIALIAS_MODE_1    0x00100 // Bit [9:8] anti-alias mode: AA enabled, resampling enabled, fetch extra lines as-needed
+#define VI_CTRL_ANTIALIAS_MODE_2    0x00200 // Bit [9:8] anti-alias mode: AA disabled, resampling enabled, operate as if everything is covered
+#define VI_CTRL_ANTIALIAS_MODE_3    0x00300 // Bit [9:8] anti-alias mode: AA disabled, resampling disabled, replicate pixels
+#define VI_CTRL_PIXEL_ADV_MASK      0x01000 // [15:12] pixel advance mode
+#define VI_CTRL_PIXEL_ADV_1         0x01000 // Bit [15:12] pixel advance mode: ?
+#define VI_CTRL_PIXEL_ADV_2         0x02000 // Bit [15:12] pixel advance mode: ?
+#define VI_CTRL_PIXEL_ADV_3         0x03000 // Bit [15:12] pixel advance mode: ?
+#define VI_CTRL_DITHER_FILTER_ON    0x10000 // 16: dither-filter mode
```

Comment:
according to current n64brew docs this is "dedither" not "dither" :shrug:

Reply:
Right, what it means by "dither filter" is a filter that smooths out dithering applied by earlier stages rather than a filter that performs dithering. Unfortunately, that's just how SGI named it.

Reply:
I forgot/didn't notice some of these are original names okay